lego/apps/events/serializers/events.py

```diff
@@ -192,6 +192,23 @@ def get_attended_count(self, event):
         return event.registrations.filter(presence=PRESENCE_CHOICES.PRESENT).count()
 
 
+class ImageGallerySerializer(BasisModelSerializer):
+    cover = ImageField(source="key", required=True, options={"height": 500})
+
```

Comment:
As with the image fields in other serializers, I would suggest also adding a `placeholder` that can plug into the image component on the frontend for smoother loading. You can just copy what's done in the serializers over here :).